Libraries/Text/RCTTextAttributes.m

```diff
@@ -95,27 +95,27 @@ - (NSParagraphStyle *)effectiveParagraphStyle
         alignment = NSTextAlignmentRight;
       }
     }
-    
```

Comment:
Its not a big deal, but generally for the sake of diff'ing when merging from upstream, I try to keep the source identical to upstream wherever possible: even if its just whitespace or spelling mistakes in comments.

Reply:
I had to add [`trim_trailing_whitespace`](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#trim_trailing_whitespace) locally to `.editorconfig` to fix the issue.